docs/submodule-update.md

```diff
@@ -0,0 +1,11 @@
+
+In order to update the common submodule, we need to pull the latest version.
+
+```
+ cd common
+ git checkout dev
```

Comment:
We might say "`dev` or whatever branch/commit is reqd" (since this value might not be `dev` in the case of a release or if there are experimental changes on a branch in common that you want to pick up in MSAL)

Reply:
Updated.